torchrl/collectors/collectors.py

```diff
@@ -550,6 +550,7 @@ def __init__(
         else:
             self.policy_weights = TensorDict({}, [])
 
+        print("sending env to device", self.device)
```

Comment:
Maybe enclose this in a `if verbose:` like the other print statements?

Reply:
Oooops 😅